scripts/rules.matterlint

```diff
@@ -79,9 +79,25 @@ all endpoints {
 
 endpoint 0 {
   // Identifiers for clusters are loaded from XML files
+  // The required clusters are from the spec for RootNode (section 2.1.5 at the moment):
+
   require server cluster Basic;
+  require server cluster AccessControl;
   require server cluster GroupKeyManagement;
-  require server cluster NetworkCommissioning;
   require server cluster GeneralCommissioning;
-  require server cluster OTASoftwareUpdateRequestor;
+  require server cluster AdministratorCommissioning;
+  require server cluster OperationalCredentials;
+  require server cluster GeneralDiagnostics;
+
+  // Required only if !CustomNetworkConfig. For now assume yes for matter examples
+  require server cluster NetworkCommissioning;
```

Comment:
Please do not add this rule, otherwise real IP-only examples cannot be tested

Reply:
Updated to comment this out.
The linter loads rules files (with this being the default) so eventually we may want a "examples" lint that runs stricter rules for matter example code.